src/clj/rems/api/services/util.clj

```diff
@@ -0,0 +1,46 @@
+(ns rems.api.services.util
+  (:require [clojure.test :refer [deftest is testing]]
+            [rems.db.roles :as roles]
+            [rems.db.users :as users]))
+
+(defn forbidden-organization? [user-id organization]
+  (let [user (users/get-user user-id)]
+    (when (and (contains? (roles/get-roles user-id) :organization-owner)
```

Comment:
hmmm I think we need to allow owners to create anything, so perhaps: `(and (not (contains? roles :owner)) ...)`

Reply:
Good point, done.